torchtext/datasets/yelpreviewpolarity.py

```diff
@@ -33,22 +33,28 @@
 
 @_add_docstring_header(num_lines=NUM_LINES, num_classes=2)
 @_create_dataset_directory(dataset_name=DATASET_NAME)
-@_wrap_split_argument(('train', 'test'))
+@_wrap_split_argument(("train", "test"))
 def YelpReviewPolarity(root: str, split: Union[Tuple[str], str]):
     if not is_module_available("torchdata"):
         raise ModuleNotFoundError("Package `torchdata` not found. Please install following instructions at `https://github.com/pytorch/data`")
 
     url_dp = IterableWrapper([URL])
 
-    cache_dp = url_dp.on_disk_cache(
+    cache_compressed_dp = url_dp.on_disk_cache(
         filepath_fn=lambda x: os.path.join(root, _PATH),
-        hash_dict={os.path.join(root, _PATH): MD5}, hash_type="md5"
+        hash_dict={os.path.join(root, _PATH): MD5},
+        hash_type="md5"
     )
-    cache_dp = GDriveReader(cache_dp).end_caching(mode="wb", same_filepath_fn=True)
-    cache_dp = FileOpener(cache_dp, mode="b")
+    cache_compressed_dp = GDriveReader(cache_compressed_dp).end_caching(mode="wb", same_filepath_fn=True)
+    cache_compressed_dp = FileOpener(cache_compressed_dp, mode="b")
```

Comment:
For consistency, can we move FileOpener after creating on_disk_cache datapipe like done for AmazonReviewPolarity (https://github.com/pytorch/text/blob/0f7f859e412fba4a31852c1a84801a182e636fde/torchtext/datasets/amazonreviewpolarity.py#L49)

Reply:
Done!

Reply:
Just wondering, is there any benefit to calling the `FileOpener` datapipe before calling `end_caching`? Or is this purely for consistency?